front-end/pages/index.js

```diff
@@ -83,4 +83,4 @@ export default function Home() {
       </footer>
     </div>
   );
-}
+}
```

Comment:
I am not sure why this is marked as a change lmao, but it should be fine because that page will definitely be override in the future.